.github/workflows/release.yml

```diff
@@ -0,0 +1,82 @@
+name: Cli Release
+on:
+  workflow_dispatch:
+    inputs:
+      change-log:
+        description: 'Open API Changelog, Please provide as a string.'
+        required: true
+      version-type:
+        description: 'Version to upgrade, Major: 0, Minor:1, Patch: 2'
+        required: true
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x]
+    steps:
+      - name: Checkout to cli
+        uses: actions/checkout@v2
+      - name: NPM Install
+        run: npm install
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
+      - name: Run tests
+        run: |
+          npm ci
```

Comment:
Why is `npm ci` required here? Shouldn't `npm install` in above step be suffice? 

Ref: https://docs.npmjs.com/cli/v7/commands/npm-ci

Reply:
Yes,` npm install` or `npm ci` one will be enough. Will remove it.